### PR TITLE
chore: update major bc changes references

### PR DESCRIPTION
**Pull request by shortcuts**
## 🧭 What and Why

🎟 JIRA Ticket: -

### Changes included:

observed in https://github.com/algolia/AlgoliaWeb/pull/22965

the ruby client had a different naming so we can add it to the table as well

### CODE REVIEW

File: specs/major-breaking-changes-rename.json [hunk 1]
```diff
@@ -56,6 +57,7 @@
     "index.deleteObjects": "client.deleteObjects",
     "index.deleteRule": "client.deleteRule",
     "index.deleteSynonym": "client.deleteSynonym",
+    "index.exists": "client.indexExists",
```
Comment (millotp) [inline specs/major-breaking-changes-rename.json:60]: https://github.com/algolia/api-clients-automation/pull/4010

Comment (shortcuts) [reply specs/major-breaking-changes-rename.json:60]: cc @kai687 can we just not generate the link for it? it's still great to let users that can use an LSP/IDE benefit from the inline documentation

Comment (shortcuts) [reply specs/major-breaking-changes-rename.json:60]: cc @kai687

Comment (shortcuts) [inline specs/major-breaking-changes-rename.json:60]:
```suggestion
```

**Review by Fluf22 — APPROVED** [review]
✅

**Pull request merged**